inference/server/export.py

```diff
@@ -219,19 +226,68 @@ def parse_args() -> argparse.Namespace:
         type=int,
         help="Seed for the anonymizer. If not specified, no anonymization will be performed.",
     )
-    # TODO: filters: reported, score, user ID, chat ID, etc
-    # TODO: date range?
+    parser.add_argument(
+        "--from-date",
+        type=parse_date,
+        help="Only export chats created on or after this date. Format: YYYY-MM-DD",
+    )
+    parser.add_argument(
+        "--to-date",
+        type=parse_date,
+        help="Only export chats created on or before this date. Format: YYYY-MM-DD",
+    )
+    parser.add_argument(
+        "--user-id",
+        type=str,
+        help="Only export chats created by this user.",
+    )
+    parser.add_argument(
+        "--chat-id",
+        type=str,
+        help="Only export this chat.",
+    )
+    parser.add_argument(
+        "--nonzero-assistant-score",
+        action=argparse.BooleanOptionalAction,
+        help="Only export chats with at least one assistant message with score != 0.",
+    ),
+    parser.add_argument(
+        "--reported",
+        action=argparse.BooleanOptionalAction,
```

Comment:
Could we use `action="store_true"` for these filters? Would just keep it consistent with other boolean args for this and other scripts